src/main/kotlin/io/github/mojira/arisa/modules/ConfirmParentModule.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import arrow.core.right
+
+class ConfirmParentModule(
+    private val confirmationStatusWhitelist: List<String>,
+    private val targetConfirmationStatus: String
+) : Module<ConfirmParentModule.Request> {
+    data class Request(
+        val confirmationStatus: String?,
+        val linked: Double?,
+        val setConfirmationStatus: (String) -> Either<Throwable, Unit>
+    )
+
+    override fun invoke(request: ConfirmParentModule.Request): Either<ModuleError, ModuleResponse> = with(request) {
+        Either.fx {
+            assertLinkedMoreThanZero(linked).bind()
+            assertConfirmationStatusWhitelisted(confirmationStatus, confirmationStatusWhitelist).bind()
+            setConfirmationStatus(targetConfirmationStatus).toFailedModuleEither().bind()
+        }
+    }
+
+    private fun assertLinkedMoreThanZero(linked: Double?) = if ((linked ?: 0.0) > 0.0) {
```

Comment:
I would change it to more than 3 to really make sure

Reply:
Added a new `linkedThreshold` configuration